devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java

```diff
@@ -261,4 +260,14 @@ public StdLib() {
      */
     public native String[] stringSplit(String str, String separator, boolean removeEmptyEntries);
 
+    /**
+     * Returns the length of the string.
+     * <p>
+     * Note, that the length considers UTF-16 elements as one, that means that it will return 1 for "a", "ã" and "🦆".
```

Comment:
We should not mention UTF-16 here, because i'm pretty sure that the C# NeoVM implementation uses UTF-8 encoding for strings on the VM.
E.g., the '🦆' is 4 bytes long, a Gregorian symbol like 'ა' is 3 bytes, 'ã' is 2 bytes, and 'a' is 1 byte long.
I've checked these lengths with `String.length()` which gives the length of the byte array and not the length in characters in the string.
These lengths match UTF-8 which has four "levels" for encoding characters -> 1, 2, 3, and 4 bytes.

@mialbu I'll adapt the documentation text. No need to do anything from your side.

Reply:
```
We should not mention UTF-16 here, because i'm pretty sure that the C# NeoVM implementation uses UTF-8 encoding for strings on the VM.
```

Yes, you're right. I think I didn't mention it in the SDK method but forgot to rephrase that one. Also, feel free to add `ა` in our int-tests.